### PR TITLE
Trev8939/format fixes script

### PR DESCRIPTION
**Pull request by TADraeseke**
Initial pass of format fixes to make sample READMEs compliant with required script formatting.

Namely:

- screenshot.png must share same name as folder containing it
- all code highlight in READMEs should be with a single set of `<code>` rather than ```<code>```
- All lists should be made with ` * ` rather than ` - `

### CODE REVIEW

File: java/blend-renderer/README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ To apply a `BlendRenderer` to a `RasterLayer`:
...
-- MapView
-- Raster
-- RasterLayer
+* ArcGISMap
+* Basemap
+* BlendRenderer
+* ColorRamp
+* MapView
+* Raster
+* RasterLayer
```
Comment (shellygill) [inline java/blend-renderer/README.md]: In the steps section below, I think step 9 has a dodgy path ('TBD')?

File: java/change-viewpoint/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Change Viewpoint
-This sample demonstrates different ways to change the ```ViewPoint``` of a map. 
+This sample demonstrates different ways to change the `ViewPoint` of a map. 
 
 ![Change Viewpoint](change-viewpoint.png)
```
Comment (shellygill) [inline java/change-viewpoint/README.md]: 'Features' heading below does not display correctly (missing space).

File: java/display-layer-view-state/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Display Layer View State
-The sample demonstrates how to view the status of the layers on the map. This is obtained from the enum value of ```LayerViewStatus```. A signal handler is set up on the map to handle the ```LayerViewStateChangedEvent``` signal, and the status text is updated when the status changes. The list of operational layers in the map are displayed. Each row in this list also has the view status corresponding to that layer. Pan or zoom the map, to view the status changes in the layers.
+The sample demonstrates how to view the status of the layers on the map. This is obtained from the enum value of `LayerViewStatus`. A signal handler is set up on the map to handle the `LayerViewStateChangedEvent` signal, and the status text is updated when the status changes. The list of operational layers in the map are displayed. Each row in this list also has the view status corresponding to that layer. Pan or zoom the map, to view the status changes in the layers.
```
Comment (shellygill) [inline java/display-layer-view-state/README.md]: 'signal handler' is incorrect terminology for Android/Java.

File: java/feature-layer-definition-expression/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 #Feature layer definition expression
```
Comment (shellygill) [inline java/feature-layer-definition-expression/README.md]: Missing space - heading does not display correctly.

File: java/feature-layer-query/README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #Feature layer query
```
Comment (shellygill) [inline java/feature-layer-query/README.md]: Missing space - heading does not display correctly.

File: java/format-coordinates/README.md [hunk 6]
```diff
@@ -2,22 +2,22 @@
...
+Coordinates can be written and formatted in different ways, for example latitude, longitude coordinates can be formatted as decimal degrees, or degrees, minutes, and seconds. This sample demonstrates how to convert a map location `Point` in WGS 1984 spatial reference to a number of different coordinate notations (decimal degrees; degrees, minutes, seconds; Universal Transverse Mercator (UTM), and United States National Grid (USNG)), by using `CoordinateFormatter`. Additionally, coodinate notation strings can be converted to a ```Point``` and shown in the map by tapping on the notation values shown and entering a coordinate in the appropriate notation.
```
Comment (shellygill) [inline java/format-coordinates/README.md]: Fix triple-tick around Point?

File: java/generate-geodatabase/README.md [hunk 7]
```diff
@@ -6,19 +6,18 @@ This sample demonstrates how to generate a geodatabase from a feature service.
 
 Zoom to any extent and click the generate button to generate a geodatabase of features from a feature service filtered to the current extent. A red bounding box graphic will display showing the extent used. A progress bar will show the job's progress. Once the geodatabase has been generated it is stored in the app's cache directory (shown in the log). The layers in the geodatabase are then added to the map.
 
-![](image1.png)
+![Generate Geodatabase App](generate_geodatabase_.png)
```
Comment (shellygill) [inline java/generate-geodatabase/README.md]: Image does not display - wrong filename.

File: java/raster-layer-file/README.md [hunk 8]
```diff
@@ -2,22 +2,20 @@
...
 ## How it works
 
 To add a `RasterLayer` as an operational layer from a local raster file:
 
-- Create a `Raster` from a raster file.
-- Create a <code>RasterLayer</code> from the `Raster`.
-- Add it as an operational layer with `map.getOperationalLayers().add(rasterLayer)`.
-
+1. Create a `Raster` from a raster file.
+1. Create a <code>RasterLayer</code> from the `Raster`.
```
Comment (shellygill) [inline java/raster-layer-file/README.md]: Why the different code tag style here?

File: kotlin/authentication-profile/README.md [hunk 9]
```diff
@@ -1,15 +1,18 @@
 # Authentication Profile
+
 The Authentication Profile sample use the `DefaultAuthenticationChallengeHandler` class to take care of showing an authentication dialog for logging into a `Portal`. Once authenticated against the portal the app displays information about the authenticated user's profile.
 
-![Authentication Profile App](auth-profile.png) 
+![Authentication Profile App](authentication -profile.png)
```
Comment (shellygill) [inline kotlin/authentication-profile/README.md]: Image not displaying - space in filename is the problem I think.

**Comment by TADraeseke** [comment]
Thanks @shellygill. Those issues should be all taken care of now!

File: java/map-loaded/README.md [hunk 10]
```diff
@@ -10,9 +11,10 @@ A listener is set up on the map to handle the `LoadStatusChangedEvent`, and the
```
Comment (shellygill) [inline java/map-loaded/README.md]: This image does not display (file name does not match).

File: java/transforms-by-suitability/README.md [hunk 11]
```diff
@@ -93,8 +95,8 @@ To download projection engine data to your device:
 
 You can use the [Android Debug Bridge (adb)](https://developer.android.com/guide/developing/tools/adb.html) tool found in **<sdk-dir>/platform-tools** to copy files to your device:
 1. Open a command prompt on your computer.
-2. Execute the ```adb push``` command to create the ```ArcGIS/samples/PEData``` directory and copy the files from your computer to the device:
-	* ```adb push <path to PEData directory on your computer> /sdcard/ArcGIS/samples/PEData```
+2. Execute the `adb push` command to create the `ArcGIS/samples/PEData` directory and copy the files from your computer to the device:
+	* ```adb push <path to PEData directory on your computer> /sdcard/ArcGIS/samples/PEData`
```
Comment (shellygill) [inline java/transforms-by-suitability/README.md]: This line is formatted incorrectly - three ticks to start, one tick to end.

**Comment by TADraeseke** [comment]
Thanks @shellygill, further fixes made as per your comments.

**Pull request merged**